src/app.ts

```diff
@@ -21,6 +25,15 @@ export function orchestration(
         ...options,
         handler: createOrchestrator(options.handler),
     });
+
+    const result: RegisteredOrchestration = (
+        input?: unknown,
+        instanceId?: string
+    ): RegisteredOrchestrationTask => {
+        return new RegisteredOrchestrationTask(functionName, input, instanceId);
+    };
+
+    return result;
 }
 
 export function entity<T = unknown>(
```

Comment:
Any thoughts on how this improvement would look like for entities?

Reply:
I do have thoughts, but it's not as straightforward as the activity and suborchestrator case, so I'm saving it for a different PR/dicussion 😛